### DISCUSSION TITLE
Remove Variable from every Sprite

### DISCUSSION THREAD

**Discussion by Scratchfangs** [ScratchAddons/ScratchAddons#648] [Ideas]
When you right-click a variable, it will show "delete variable from every sprite". This can help if you have an unnecessary variable and you have a lot of sprites with that variable.

**Comment by scratchusernamemrtbts** [comment]
did you mean with the for this sprite only variable
I think it's impossible to do that

**Reply by Scratchfangs** [reply]
No, when the variable code blocks are in a lot of sprites and when you delete it from a specific sprite, it will come right back because the variable code blocks are in different sprites. (sorry if you dont understand :P)

**Reply by GrahamSH-LLK** [reply]
> No, when the variable code blocks are in a lot of sprites and when you delete it from a specific sprite, it will come right back because the variable code blocks are in different sprites. (sorry if you dont understand :P)

~~Really? That sounds like a bug, or something brought on by the data category tweaks addon.~~  Edit: I reread that, and now I understand.

**Reply by Scratchfangs** [reply]
Will it be considered being added?

**Reply by scratchusernamemrtbts** [reply]
> Will it be considered being added?

Idk

**Reply by Endermaster3690** [reply]
> > Will it be considered being added?
> 
> Idk

I think it should be added! I gotta remove this variable in my reactor core game popping up everywhere.

**Comment by Mr-Programmer-777** [comment]
Sure. Couldn't that simply be merged into "delete variable"?

**Reply by Endermaster3690** [reply]
Yeah that sounds good.
